### PR TITLE
feat: patch.md Step 5a.7 — write PR-level hitl when no linked task

### PR DESCRIPTION
**Pull request by dodizzle**
## Summary
- `patch.md` Step 5a.7 (second-round-disagreement escalation): when `PR_TASK_ID` is empty (no linked task), the PR previously kept re-qualifying as a patch candidate forever since nothing was ever recorded to stop it.
- Now PATCHes `$SHIPWRIGHT_TASK_STORE_URL/prs/$PR_RECORD_ID` with `{"hitl": true, "blockedReason": "..."}` in that case, reusing `PR_RECORD_ID` already claimed in Step 5a.6. The non-empty `PR_TASK_ID` case (PATCH the task) is unchanged.
- Refreshed `docs/task-store.md` to document the `hitl`/`blockedReason` fields covering this escalation scenario.

## Acceptance Criteria
| # | Criterion | Status |
|---|-----------|--------|
| 1 | Empty `PR_TASK_ID` case PATCHes `/prs/$PR_RECORD_ID` with `hitl:true` + `blockedReason` | MET |
| 2 | Non-empty `PR_TASK_ID` case unchanged | MET |
| 3 | Safe to deploy standalone | MET |
| 4 | Test asserts new PATCH branch | MET |

## Test Plan
- `bun test plugins/shipwright/commands/patch.content.test.ts` — 55 pass, 0 fail
- `bun run typecheck` (plugins/shipwright) — clean
- `bunx biome lint` — clean
- [x] Pre-commit checks passing

Generated with [Claude Code](https://claude.com/claude-code)

### CODE REVIEW

File: plugins/shipwright/commands/patch.md [hunk 1]
```diff
@@ -641,8 +641,17 @@ do not reset `reviewState`.
      -d '{"hitl": true}' > /dev/null 2>&1 || \
      echo "⚠ PATCH /tasks/$PR_TASK_ID hitl flag failed — continuing"
    ```
-   If `PR_TASK_ID` is empty (no linked task on the PR record), log a warning and skip the
-   PATCH — still post the PR comment below.
+   If `PR_TASK_ID` is empty (no linked task on the PR record), PATCH the PR record itself
```
Comment (dodizzle) [inline plugins/shipwright/commands/patch.md:644]: This PATCH (hitl:true + blockedReason on the PR record) doesn't actually stop re-qualification. `agent/src/check-patch.ts`'s `getPatchCandidates` only checks `linkedTask?.hitl` (line 449) — its `PrRecord` interface (lines 90-93) never reads `record.hitl`. Same gap in `check-review.ts:149` and `check-deploy.ts:251`. There's an unused, unit-tested helper for exactly this — `isPrRecordBlockedForDispatch(pr: { hitl?: boolean })` in `check-helpers.ts:529` — with zero call sites outside its own test. This branch needs `isPrRecordBlockedForDispatch(record)` wired into the candidate providers' gating to actually work.

**Review by dodizzle — COMMENTED** [review]
Verdict: COMMENT — The new Step 5a.7 branch PATCHes hitl:true onto the PR record when there's no linked task, but no candidate provider (check-patch.ts, check-review.ts, check-deploy.ts) reads PR-record-level hitl — only linkedTask?.hitl is checked. The purpose-built isPrRecordBlockedForDispatch() helper in check-helpers.ts already does the right check but is never called from any production candidate-collection path. As written, this PR writes a field that nothing downstream reads, so the PR will keep re-qualifying for patch/review exactly as before — the change doesn't fix the spinning-forever problem it exists to solve.

**Review by dodizzle — COMMENTED** [review]
Verdict: APPROVE — Correctly closes the loop on the prior review's finding by wiring `isPrRecordBlockedForDispatch` into all three PR-record-based candidate providers (check-patch.ts, check-review.ts, check-deploy.ts). Verified insertion-point ordering is safe in all three, `release()` never clears `hitl` so the escalation sticks, and check-dev-task.ts correctly needed no change (task-based, not PR-record-based). Tests target the precise edge case, lint clean, CI green.

**Review by dodizzle — COMMENTED** [review]
Verdict: APPROVE — Re-review at the new head (merge of main, no new logic). The prior finding is fixed: `isPrRecordBlockedForDispatch(record)` is now correctly wired into `check-patch.ts`, `check-review.ts`, and `check-deploy.ts`, ordered after the `claimedBy` check in each. `release()` never clears `hitl`, so the escalation sticks. Docs match the writable-fields allowlist, all 4 acceptance criteria are met, and CI is green. No new issues in this round's delta.
